docs/en/docs/tutorial/path-params-numeric-validations.md

```diff
@@ -12,19 +12,15 @@ First, import `Path` from `fastapi`:
 
 ## Declare metadata
 
-You can declare all the same parameters as for `Query`.
+You can declare all the same parameters as for `Query` except for the default. A path parameter is always required as it has to be part of the path. You should declare it with `...` to correctly mark it as required, for convenience FastAPI already does this if you leave it out.
```

Comment:
Maybe just leave out the "You should declare it with ..." part?

Reply:
It still has meaning, we're explaining that compared to Query parameters, Path parameters are always required and that's why you should declare it with `...` to mark it as such (in relation to Query parameters). Then adding that since this is the case, for Path parameters FastAPI defaults to `...` if you leave it out.